projects/kit/components/input-time/input-time.template.html

```diff
@@ -43,3 +43,15 @@
         </button>
     </tui-data-list>
 </ng-template>
+
+<ng-template #iconContent="polymorpheus" polymorpheus>
```

Comment:
This is probably not the first time we have something like this. How about we just add context with $implicit size directly to `primitive-textfield`?